### PR TITLE
OS specific promotion of tools on /download/server and /download/desktop

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Test of OS specific promotion of tools on /download/desktop and /download/server
- Trying to improve on the multipass strip and promote up to three items based on the user's OS

## Notes

- This is a test before we get to the larger work around the promotion and is wired up with utm data

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/download/desktop and http://0.0.0.0:8001/download/server
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Change your user agent (in the inspector his Ctrl+Shift+P, type 'network conditions' and change your user agent and see the items in the 'More ways to use Ubuntu everywhere' strip change

### linux
image

### windows
image

### mac
image

### CODE REVIEW

**Comment by pmahnke** [comment]
@anthonydillon I normally say that as well, but if we think of these as ads, what would you do?

Assume we have more things to promote that we have space for?

Very open to thoughts.

  | Windows | macOS | Linux | Desktop | Server
-- | -- | -- | -- | -- | --
WSL1 | X |   |   | X |  
WSL2 | X |   |   | X |  
Bash | X |   |   | X |  
Multipass | X | X | X | X | X
LXD |   |   | X | X | X
MicroK8s | X | X | X |   | X
MicroStack | X | X | X |   | X

Desktop |   |   |   
-- | -- | -- | -- 
Windows user | WSL | Bash | Multipass 
macOS user | Multipass |   |   
Linux user | Multipass | LXD |

Server |   |   |   |     
-- | -- | -- | -- | -- 
Windows user | Multipass | MicroK8s | MicroStack |   |  
macOS user | Multipass | MicroK8s | MicroStack |   |  
Linux user | Multipass | MicroK8s | MicroStack | LXD

**Comment by anthonydillon** [comment]
I guess when you think of it as ads. It just smells like a confused messaging issue then to much to promote. We could offer tooling per OS pages or something. Like snapcrafts distro pages to advertise the products fully in an exciting way and good for SEO.

Once we have the pages we could do some linking between these pages and the OS tooling pages?

**Comment by pmahnke** [comment]
@anthonydillon I think we will have per OS pages that cover all the tools for each one.  I guess here you could just promote those pages, but I wonder how much use you would get.

We could also tab this section and start on the OS you are currently running?

**Comment by cassiocassio** [comment]
Should the headline reflect the OS, .e.g

"More ways to use Ubuntu on Windows" ?

Should the multipass or other text refer to the advantages for the targeted platform, e..g:

"On demand build and test build and test VMs for cloud-devs on Mac desktops, with brew install and update"

**Comment by wgx** [comment]
Made some changes following discussion with @pmahnke 

* Change headline to be more user-focused - "what does this thing allow me to do?"
* Add call-to-action button which includes the name of the product - users of product X may never have heard of product Y so it makes sense to put it on the button not the headline
* Rewrote copy accordingly

**Example:**

image

**Multipass in proposed style:**

image

**Proposed copy:**

`
          <h3 class="p-matrix__title"><a href="https://linuxcontainers.org">Containers on Linux</a></h3>
          <p class="p-matrix__desc">
            A pure-container hypervisor. Replace legacy app VMs with containers for speed and density with LXC.
          </p>

          <h3 class="p-matrix__title"><a href="https://multipass.run">Instant Ubuntu VMs</a></h3>
          <p class="p-matrix__desc">
            On-demand build-and-test VMs for cloud devs on Windows, Mac and Linux desktops with Multipass.
          </p>

          <h3 class="p-matrix__title"><a href="https://microk8s.io/">Small, Fast Kubernetes.</a></h3>
          <p class="p-matrix__desc">
            MicroK8s is small, fast and fully-conformant Kubernetes. Made for developers and devops. Great for edge and IoT.
          </p>

          <h3 class="p-matrix__title"><a href="https://ubuntu.com/wsl">Ubuntu on Windows 10</a></h3>
          <p class="p-matrix__desc">
            Install a complete Ubuntu terminal environment in minutes on Windows 10 with Windows Subsystem for Linux (WSL).
          </p>
`


cc @anthonydillon

**Comment by pmahnke** [comment]
This is looking good.  I think we need to update the grid and/or add ubunut.com/appliance as well.

**Comment by wgx** [comment]
@pmahnke  Appliance copy:

### Images for your PC or Raspberry Pi

**Transform a spare PC or Pi into a smart, secure, and ultra-simple box that does exactly what you want.**

CTA: Browse Appliances
Href: https://ubuntu.com/appliance

File: templates/download/desktop/index.html [hunk 1]
```diff
@@ -4,25 +4,25 @@
 {% block meta_copydoc %}https://docs.google.com/document/d/1ScuTtdrm3-iAvizTDxksVTAerV6QkCUN8Sg60uUlJsk/edit{% endblock meta_copydoc %}
 
 {% block content %}
-<section class="p-strip--suru-topped is-bordered">
+<section class="p-strip--suru-topped">
   <div class="u-fixed-width">
     <h1>Download Ubuntu Desktop</h1>
```
Comment (anthonydillon) [inline templates/download/desktop/index.html:9]: The server page as `u-no-margin--bottom` on it.

**Pull request merged**